content/tutorials/grafana-fundamentals.md

```diff
@@ -345,6 +345,15 @@ Once you've acknowledged an alert, consider pausing it. This can be useful to av
 1. Find your alert in the list, and click the **Pause** icon on the right. The **Pause** icon turns into a **Play** icon.
 1. Click the **Play** icon to resume evaluation of your alert.
 
+### Clean up the local environment
+
+The tutorial has running Docker containers.
+When you want to clean up this local environment such that there are no
+running Docker containers,
+use `docker ps` to acquire the IDs of the tutorial's containers.
```

Comment:
```suggestion
use `docker ps` to acquire the IDs of the tutorial containers.
```

Reply:
Is this a Grafana style thing, @oddlittlebird ? Because the way all these instances were originally is a very normal way to refer to containers like these. The possessive form isn't strictly necessary.

Reply:
Are you two suggesting the exact same thing? :)